software/MainboardCommunication.py

```diff
@@ -0,0 +1,17 @@
+import struct
+import serial
+import time
+
+disable_failsafe = 0
+speed1 = 20
+speed2 = 0
+speed3 = -20
+thrower_speed = 00
+
+andmed = struct.pack('<hhhHBH', speed1, speed2, speed3, thrower_speed, disable_failsafe, 0xAAAA)
```

Comment:
Use English. The course is in English and we have foreign students taking the course who might want read and understand the code.